Assets/AnkrSDK/Examples/Scripts/EventListenerExample/EventListenerExample.cs

```diff
@@ -11,10 +11,21 @@
 
 namespace AnkrSDK.EventListenerExample
 {
+	/// <summary>
+	/// Your usual flow will be this:
+	/// 	Get the subscriber instance with provided Endpoint
+	///			_eventSubscriber = ankrSDK.CreateSubscriber(ERC20ContractInformation.WsProviderURL);
+	///		Manually connect socket
+	///			_eventSubscriber.ListenForEvents().Forget();
+	///		Subscribe custom handler for events
+	///			_eventSubscriber.OnOpenHandler += UniTask.Action(SubscribeWithTopics);
+	///		Unsubscribe
+	///			_eventSubscriber.Unsubscribe(_subscription).Forget();
+	///		Manually stop listen
+	///			_eventSubscriber.StopListen();
+	/// 	</summary>
 	public class EventListenerExample : UseCase
```

Comment:
Not from this commit but public async UniTaskVoid SubscribeWithTopics() could be made private